app/controllers/courses/age_range_controller.rb

```diff
@@ -43,6 +27,44 @@ def update
 
   private
 
+    def update_age_range_param
+      if age_from_param.present? && age_to_param.present? && age_range_is_other?
+        params[:course][:age_range_in_years] = "#{age_from_param}_to_#{age_to_param}"
+      elsif age_range_is_other?
+        params[:course][:age_range_in_years] = nil
+      end
```

Comment:
While we're here, is this sufficiently better to be worth changing it to:

```
      if age_range_is_other?
        if age_from_param.present? && age_to_param.present?
          params[:course][:age_range_in_years] = "#{age_from_param}_to_#{age_to_param}"
        else
          params[:course][:age_range_in_years] = nil
        end
      end
```

Reply:
@misaka ... yeah open on this I. I tend to avoid nested if / else statements but happy to go with whatever people think is most readable. An another alternative could be to extract `age_from_param.present? && age_to_param.present? && age_range_is_other?` to it's own method e.g (`custom_age_range?`)